src/features/rewards/mobile/mainToggleMobile/style.ts

```diff
@@ -30,26 +30,27 @@ export const StyledLeft = styled<{}, 'div'>('div')`
 `
 
 export const StyledRight = styled<{}, 'div'>('div')`
-  padding: 0px 15px;
-  display: flex;
   height: 66px;
-  align-items: center;
-  margin-top: -3px;
-
-  @media (max-width: 355px) {
-    width: 55px;
-    position: relative;
-    right: 45px;
-  }
+  position: fixed;
```

Comment:
This seems weird to have the checkbox fixed position on a header that is already fixed position. If the aim is to vertically center the checkbox then, since the parent is flex, it can be achieved via:
```css
align-self: center;
margin-right: 15px;
```
Although preferably the parent would specify a padding instead of the element individually specifying its right margin.